### PR TITLE
add integration for Landkreis Rhön Grabfeld

### PR DESCRIPTION
**Pull request by Random-Cow**
Add integration for Landkreis Rhön Grabfeld.
Tested via test-script and also runs perfectly fine on my local home-assistant instance

### CODE REVIEW

File: doc/source/landkreis_rhoen_grabfeld.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Abfallkalender Würzburg
```
Comment (mampfes) [inline doc/source/landkreis_rhoen_grabfeld.md:1]: Wuerzburg? --> Copy-Paste Error???

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/landkreis_rhoen_grabfeld.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+    "District only": {"district": "Oberwaldbehrungen"},
+    "empty": {}
+}
+
+
+class Source:
+    def __init__(self, city: str = None, district: str = None):
+        self._city = city
+        self._district = district
+        self._iconMap = {
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/landkreis_rhoen_grabfeld.py:21]: Please make icon-map a global variable (like TEST_CASES).

File: custom_components/waste_collection_schedule/waste_collection_schedule/source/landkreis_rhoen_grabfeld.py [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+            "stadt": self._city,
+            "ortsteil": self._district
+        })
+
+        r.raise_for_status()
+
+        entries = []
+        for event in r.json():
+            # filter out Sammelstellen, Wertstoffhof and Wertstoffzentrum
+            if event["muellart"] in self._iconMap:
```
Comment (mampfes) [inline custom_components/waste_collection_schedule/waste_collection_schedule/source/landkreis_rhoen_grabfeld.py:40]: Not sure if it is a good idea to use the icon-map to filter out waste types. I think I would prefer a blacklist.

**Review by mampfes — CHANGES_REQUESTED** [review]

**Comment by mampfes** [comment]
Please add the new source to info.md also.

File: doc/source/landkreis_rhoen_grabfeld.md (hunk 1 above)
Comment (Random-Cow) [reply doc/source/landkreis_rhoen_grabfeld.md]: Yeah, copy-paste error...

**Pull request merged**